cg/server/api.py

```diff
@@ -359,7 +359,7 @@ def options():
             for customer in customers
         ],
         applications=apptag_groups,
-        panels=[panel.abbrev for panel in db.panels()],
+        panels=[panel.abbrev for panel in db.get_panels()],
```

Comment:
```suggestion
        panel_abbreviations=[panel.abbrev for panel in db.get_panels()],
```

Reply:
This is a json response, changing the name would involve chasing down all callers of this endpoint and change the expected format.